examples/convert_group_studies.py

```diff
@@ -100,3 +100,8 @@
 # Now let's see the structure of the BIDS folder we created.
 
 print_dir_tree(output_path)
+
+###############################################################################
+# Now let's generate a report on the dataset.
+dataset_summary = create_methods_paragraph(bids_root=output_path)
+print(dataset_summary)
```

Comment:
nitpick: consider using [textwrap](https://docs.python.org/3/library/textwrap.html) for pretty printing this

![image](https://user-images.githubusercontent.com/9084751/85435440-bdef2400-b587-11ea-8bf8-918bb654a7ba.png)